pkg/project/project.go

```diff
@@ -2,12 +2,19 @@ package project
 
 var (
 	description = "chart-operator is our agent for deploying Helm charts as releases."
-	name        = "chart-operator"
 	gitSHA      = "n/a"
+	name        = "chart-operator"
 	source      = "https://github.com/giantswarm/chart-operator"
-	version     = "n/a"
+	version     = "1.0.0-alpha.1-dev"
```

Comment:
This is a bit of a monster but we created a alpha1 release to make testing easier.